src/main/scala/dpla/ingestion3/enrichments/StringUtils.scala

```diff
@@ -61,8 +61,41 @@ object StringUtils {
       value.split(delimiter).map(_.trim).filter(_.nonEmpty)
     }
 
-    val stripEndingPunctuation: SingleStringEnrichment = {
-      value.replace("""[^\w\'\"\s]+$""", "")
+    /**
+      * Removes leading and trailing: colons, semi-colons, commas
+      * slashes, hyphens and whitespace characters (whitespace, tab, new line and line feed)
+      *
+      */
+
+    val cleanupEndingPunctuation: SingleStringEnrichment = {
+      // FIXME this is going to be problematic if the last character
+      // is valid but not alphanumeric (e.g. ends with a " or ')
```

Comment:
Do you want to address this `FIXME` in this PR or a later PR?

Reply:
@guegueng is working on that list of characters.